selinux/qubes-gui-agent.te

```diff
@@ -0,0 +1,14 @@
+policy_module(qubes-gui-agent, 0.0.1)
+
+require {
+	type xdm_t, etc_t, pulseaudio_t;
+	class passwd rootok;
+}
+
+allow xdm_t etc_t:file { create write };
+allow xdm_t self:passwd rootok;
+pulseaudio_domtrans(xdm_t)
+dev_rw_xen(xdm_t)
+init_domtrans_script(xdm_t)
+dev_create_sysfs_files(xdm_t)
```

Comment:
Is this about `echo 1073741824 > /sys/module/xen_gntalloc/parameters/limit`, or something else? We could move this to somewhere else (`ExecStartPre` in systemd unit?).

Reply:
Presumably, yes.